src/main/java/core/basesyntax/strategy/SupplyOperationImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+
+public class SupplyOperationImpl implements OperationHandler {
```

Comment:
```suggestion
public class SupplyOperationHandler implements OperationHandler {
```
same for other handlers - just better naming